package/utils/px5g/Makefile

```diff
@@ -19,6 +19,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/px5g/Template
   SECTION:=utils
   CATEGORY:=Utilities
+  SUBMENU:=Encryption
```

Comment:
@blogic - "Encryption" is fine with me. I couldn't come up with something better and after all it generates a cert pair for encryption